worker/src/email/resources/sql/enqueue-messages-email.sql

```diff
@@ -19,8 +19,8 @@ BEGIN
       m.campaign_id = selected_campaign_id
       -- enqueue only those that have not been enqueued - this means that when we retry, we will have to set dequeued_at to null
```

Comment:
amend this comment since the setting of dequeued_at to null now occurs in the logging step